ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -103,6 +103,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-sdn-workers-rhcos-rhel8
+- as: e2e-aws-ipi-disconnected-private-sno-baselinecaps-none-p2
+  cron: 0 2 */7 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @disconnected and @network-ovnkubernetes and @singlenode
+        and not @fips'
+      TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&
+    test:
+    - chain: openshift-e2e-test-qe
```

Comment:
does it mean component team should update their own logic to exclude these component which is disabled in this profile?

Reply:
cucumber use tags to filter tests. As we didn't add specific tags for sub-team and/or specific features, cucushift can not skip specific tests for sub-team/features.
We can use TEST_FILTERS to skip specific features, but I need the exact filter for specific teams/features, or feel free to use follow up PRs to add extra filters to exclude specific tests.